sys/include/net/gnrc/rpl/structs.h

```diff
@@ -279,9 +289,21 @@ typedef struct {
      */
     int (*parent_cmp)(gnrc_rpl_parent_t *parent1, gnrc_rpl_parent_t *parent2);
     gnrc_rpl_dodag_t *(*which_dodag)(gnrc_rpl_dodag_t *, gnrc_rpl_dodag_t *); /**< compare for dodags */
-    void (*reset)(gnrc_rpl_dodag_t *);    /**< resets the OF */
+
+    /**
+     * @brief Reset the state of the objective function.
+     *
+     * @param[in]   dodag   RPL dodag object.
+     */
+    void (*reset)(gnrc_rpl_dodag_t *dodag);
     void (*parent_state_callback)(gnrc_rpl_parent_t *, int, int); /**< retrieves the state of a parent*/
-    void (*init)(void);  /**< OF specific init function */
+
+    /**
+     * @brief Initialize the objective function.
+     *
+     * @param[in]   dodag   RPL dodag object.
+     */
+    void (*init)(gnrc_rpl_dodag_t *dodag);  /**< OF specific init function */
```

Comment:
please remove inline comment behind, if info is required move up below `@brief` - having both is weird

Reply:
aaand it's gone.